server/api.js

```diff
@@ -34,4 +35,10 @@ apiRouter.use('/lob', lob)
 apiRouter.use('/checkout', checkout)
 //apiRouter.use('/twilio', twilio)
 
+// Add the mock Cicero API if we're in development
```

Comment:
I do not see how `mockRouter` will ever be called by the front end. Check out Express middleware: https://expressjs.com/en/guide/using-middleware.html. You could use this to intercept the call to representatives and redirect to `mockRouter`.

Not that you have to do this, but I think a really good solution would be to ditch json-server and create you own routing module following the pattern set by the other routers. That way, you go through creating an endpoint, and writing integration tests for it 🙂